app/sidekiq/asset_manager_attachment_metadata_worker.rb

```diff
@@ -6,10 +6,6 @@ def perform(attachment_data_id)
 
     return if attachment_data.blank?
 
-    # this call needs to be here to run when we do an edition discard,
-    # unless we add a custom service for that
```

Comment:
Just to make the commit sequence easier to follow, I'd consider tweaking the inline comment to something along the lines of "Temporarily calling this here until we add a custom service to do this - see next commit"